Makefile.am

```diff
@@ -241,22 +241,24 @@ endif
 # -------
 
 netbsd_platform_headers = \
+	generic/gettime.h \
+	generic/hostname.h \
+	generic/uname.h \
 	netbsd/Platform.h \
 	netbsd/ProcessField.h \
 	netbsd/NetBSDProcessList.h \
-	netbsd/NetBSDProcess.h \
-	generic/hostname.h \
-	generic/uname.h
+	netbsd/NetBSDProcess.h
```

Comment:
Should sort before `NetBSDProcessList.h`

Reply:
Should be fixed in 0fd67f9